src/course/syllabus/pre-apprenticeship/testing/schedule.md

```diff
@@ -1,72 +1,70 @@
 Learn how to make sure your code works correctly by creating automated tests.
 
-## Week 1
+## Week 7
 
-### Monday
+### Session 1
 
 - 17:00-17:15 Check-in
-- 17:15-17:45 [Intro talk](https://fac-slides.netlify.app/slides/testing/)
-- 17:45-18:15 [Build a testing library](/workshops/learn-testing/)
-- 18:15-19:45 [Unit testing workshop](/workshops/learn-unit-testing/)
+- 17:15-18:15 Peer-led tutorial groups
+- 18:15-19:00 Class discussion & solutions
+- 19:00-19:45 Start on [Integration testing workshop](/workshops/learn-integration-testing/)
```

Comment:
@oliverjam alone. I think we are both strongly in favour or rebalancing the programme to allow more space for individual study.

Reply:
In truth, this is a bit of a placeholder. We may eventually want to use the time differently, but for now this is a reasonable transition to a slightly different way of doing things. I don't mind you pairing people up if you think this would be useful exercise, though.